### PR TITLE
Script to run functional tests

### PR DESCRIPTION
**Pull request by andybondar**
Added script to run functional tests from Jenkins job(s), please review

### CODE REVIEW

**Comment by roman-verchikov** [comment]
Please move script to `devlab/jenkins` folder

File: devlab/scripts/run_functional_tests.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+set -x
+
+trap 'clean_exit $LINENO $BASH_COMMAND; exit' SIGHUP SIGINT SIGQUIT SIGTERM EXIT
+clean_exit()
+{
+    pushd $WORKSPACE/cloudferry/devlab
```
Comment (realsystem) [inline devlab/scripts/run_functional_tests.sh]: $WORKSPACE is not defined

Comment (andybondar) [inline devlab/scripts/run_functional_tests.sh]: >Please move script to devlab/jenkins folder

Done

Comment (andybondar) [inline devlab/scripts/run_functional_tests.sh]: Added exception:

```
if [ -z "$WORKSPACE" ]; then
    echo "'WORKSPACE' is not defined"
    exit 1
fi
```

Comment (realsystem) [inline devlab/scripts/run_functional_tests.sh]: But in this case we can't run this script from devlab manually. It's a simple to define it if not defined.

File: devlab/jenkins/run_functional_tests.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+echo "Preparing environment"
+
+pushd $WORKSPACE/cloudferry
+
+#git remote update
+# cloudferry source dir is not deleted after job finish, so if
+# pull request cannot be automatically rebased, we must abort it explicitly and 
+# exit with failure. In case rebase succeeded, functional test should
+# move on.
+#git pull --rebase origin devel || ( git rebase --abort && exit 1 )
```
Comment (roman-verchikov) [inline devlab/jenkins/run_functional_tests.sh]: Let's remove these lines, I dont think we need them

File: devlab/jenkins/run_functional_tests.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [ -z "$WORKSPACE" ]; then
+    echo "'WORKSPACE' is not defined"
+    exit 1
```
Comment (realsystem) [inline devlab/jenkins/run_functional_tests.sh]: Once again ... in this case we can't run this script from devlab manually. It's simple to define it if not defined.

**Comment by realsystem** [comment]
+1

**Comment by miarmak** [comment]
Squash these 3 commits into 1, please. And also use this pattern for commit description:
```
<Commit title, no longer than 79 symbols>

<Multiline commit description>
```

**Comment by realsystem** [comment]
+1, but squash is needed

**Comment by Settis** [comment]
Looks good +1

**Comment by realsystem** [comment]
+1

File: devlab/jenkins/run_functional_tests.sh [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    vagrant destroy --force
+    popd
+}
+
+echo "Preparing environment"
+
+pushd $CF_DIR
+
+if [ "$JOB_NAME" = "cloudferry-functional-tests" ]; then
+    git checkout devel
```
Comment (andybondar) [inline devlab/jenkins/run_functional_tests.sh]: remove this string

**Pull request merged**